app/views/search/_granule_search.html.erb

```diff
@@ -78,7 +78,7 @@
           <span data-bind="visible: !cmrGranuleQuery.validateCloudCoverRange(cmrGranuleQuery.cloudCover.min(),cmrGranuleQuery.cloudCover.max())" class="error">Minimum must be less than Maximum</span>
         </p>
       </dd>
-      <!-- ko if: $root.labs() && searchable_attributes -->
+      <!-- ko if: ($root.labs() || has_feature('<%=Rails.configuration.cmr_tag_namespace.gsub(/\.$/, '')%>', 'attribute_search')) -->
```

Comment:
The && and || are still wrong. Now if you have the labs parameter set, the heading will show up even if there are no searchable attributes.

Also, Can you avoid passing the cmr_tag_namespace in erb and just get it from Javascript?  See note I'll be making above.